packages/std/src/ibc.rs

```diff
@@ -48,6 +48,18 @@ pub enum IbcMsg {
     /// This will close an existing channel that is owned by this contract.
     /// Port is auto-assigned to the contract's IBC port
     CloseChannel { channel_id: String },
+    /// This writes the ibc acknowledgement for the given packet to the chain.
+    /// This is only needed if you need to delay the acknowledgement.
+    /// In this case, you can return an [`IbcReceiveResponse`] with an acknowledgement of `None`
+    /// from `ibc_packet_receive` and call this message later.
+    WriteAcknowledgement {
+        /// The channel on the destination chain (the chain this is executed on)
+        /// to write the acknowledgement to.
+        dest_channel: IbcEndpoint,
```

Comment:
We can use an API similar to SendPacket:

```suggestion
        /// The local channel (on the chain this is executed on)
        /// to write the acknowledgement to.
        channel_id: String,
```